packages/-build-infra/package.json

```diff
@@ -21,7 +23,20 @@
     "ember-cli-string-utils": "^1.1.0",
     "git-repo-info": "^2.0.0",
     "npm-git-info": "^1.0.3",
-    "silent-error": "^1.1.1"
+    "silent-error": "^1.1.1",
+    "glob": "^7.1.3",
+    "mocha": "^6.1.2",
+    "mocha-only-detector": "1.0.1",
```

Comment:
We should make an issue to remove this (in favor of using an eslint plugin that fails for `.only`).

Reply:
https://github.com/emberjs/data/issues/6109